components/kserve/kserve.go

```diff
@@ -59,11 +66,31 @@ func (d *Kserve) ReconcileComponent(owner metav1.Object, cli client.Client, sche
 		return err
 	}
 
-	err = deploy.DeployManifestsFromPath(owner, cli, ComponentName,
+	if err := deploy.DeployManifestsFromPath(owner, cli, ComponentName,
 		Path,
 		namespace,
-		scheme, enabled)
-	return err
+		scheme, enabled); err != nil {
+		return err
+	}
+
+	err = common.UpdatePodSecurityRolebinding(cli, []string{"odh-model-controller"}, namespace)
+	if err != nil {
+		return err
+	}
+
+	// Update image parameters
+	if err := deploy.ApplyImageParams(Path, dependentImageParamMap); err != nil {
+		return err
+	}
+
+	if err := deploy.DeployManifestsFromPath(owner, cli, ComponentName,
+		DependentPath,
+		namespace,
```

Comment:
Is this expected to be deployed in the monitoring namespace if it differs from `applicationNamespace`? If yes, you would need to update the namespace similar to modelmesh https://github.com/opendatahub-io/opendatahub-operator/blob/main/components/modelmeshserving/modelmeshserving.go#L74-L92

Reply:
No, this is expected to be deployed in the `applicationsNamespace` along with the Kserve Controller